docs/reference/buildx_history_logs.md

```diff
@@ -14,3 +14,34 @@ Print the logs of a build
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Print the logs for a completed build. The output appears in the same format as `--progress=plain`, showing the full logs for each step without multiplexing.
+
+By default, this shows logs for the most recent build on the current builder.
```

Comment:
Could we also show offset variant. `'^1'` would be one before most recent and so on. This is the same for other commands that take build ref as an argument.